CDCgov/cdc-open-viz
# Enhancement [DEV-9945]: Multi-series Box Plot - Data Table Updates

## [Replace With Ticket Number]



## Testing Steps
Open Box Plot
Add more series
The table should be updated and handle more series as shown on the image attached
<img width="1496" alt="Screenshot 2024-12-04 at 12 43 48" src="https://github.com/user-attachments/assets/c6dd645e-14ab-4533-9385-d8ed7b606861">




## Self Review

- I have added testing steps for reviewers
- I have commented my code, particularly in hard-to-understand areas
- My changes generate no new warnings
- New and existing unit tests are passing

## Screenshots (if applicable)



## Additional Notes
